src/utils/data.ts

```diff
@@ -0,0 +1,88 @@
+export const resourcesData = [
+  {
+    title: "Core concept of collaborative Design Process",
```

Comment:
These data should come from the json in the public folder. we shouldn't hardcode any more data into the project

Reply:
the only data we are using is description just becouse they do not exist in json file,
the description on yellow card is not there.

Reply:
In this images I highlighted the information that we do not have in the json file that is why I created them

![Screenshot Capture - 2024-04-11 - 18-16-57](https://github.com/EnAccess/co-design-FE/assets/77101006/3620a0c2-75a6-40ba-a976-86fc825a46ef)



![Screenshot Capture - 2024-04-11 - 18-09-21](https://github.com/EnAccess/co-design-FE/assets/77101006/237dbf36-2328-41f7-9135-bbf0c9c40a6b)